src/uct/rocm/rocm_cma_ep.c

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (C) Advanced Micro Devices, Inc. 2016 - 2017. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+
+#include "rocm_cma_ep.h"
+#include "rocm_cma_md.h"
+#include "rocm_common.h"
+
+/* Include HSA Thunk header file for CMA API*/
+#include <hsakmt.h>
+
+#include <uct/base/uct_log.h>
+#include <ucs/debug/memtrack.h>
+#include <uct/sm/base/sm_iface.h>
+
+
+static UCS_CLASS_INIT_FUNC(uct_rocm_cma_ep_t, uct_iface_t *tl_iface,
+                            const uct_device_addr_t *dev_addr,
```

Comment:
Thanks a lot.  

I read CodeStyle and IMHO it would be great to have example of coding  because previously I interpreted it a little different :-(.

"indent function arguments on column".  So it is ok  not to "put" / indent / align them  on "4 spaces" ?

BTW:  "indent structure fields on column" - should column be 4 aligned in this case?